content/api/transmissions.apib

```diff
@@ -169,7 +169,7 @@ The template language is supported in `from`, `subject`, `text`, `html`, `amp_ht
     + from (enum, required) - Email address used to compose the email's `From` header. The domain must be a verified sending domain. Can be a string with a valid email address or an object with the following attributes:
         + (string)
         + (object)
-            + name (string) - The name associated with the email address.
+            + name (string) - The name associated with the email address (special characters must be escaped).
```

Comment:
I feel like this needs some examples, because it's not entirely clear from this what you should do with quoted characters.

From [https://tools.ietf.org/html/rfc2822#section-3.2.2] the text that needs quoting appears surrounded in double quotes, and then has the format:

```
quoted-pair     =       ("\" text) / obs-qp
```

So naïvely you might expect to do something like this in the JSON:

```
"name": "backslash quoted: \\"
```

but the original customer report says we need to double-backslash escape things in our system:

```
"name": "backslash \\\\"
```

This is a little bit confusing if you're expecting the JSON to only contain escaping per RFC 2822.

I think a more concrete example somewhere in our docs would be really helpful. (It's worth noting I haven't actually investigated the original problem, or tried escapes in the `name` field.)